features/step_definitions/checks_for_vulnerable_gems_steps.rb

```diff
@@ -7,7 +7,7 @@ def gem_freshness_output
     prep = "cd spec/fixtures/#{@projectname}/ && cp Gemfile_lock Gemfile.lock"
     `#{prep} && bundle-audit check; rm -f Gemfile.lock`
       .split("\n")
-      .reject { |line| line.match('patched versions found') }
+      .reject { |line| line.match('ulnerabilities found') }
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.